lib/ssdp/consumer.rb

```diff
@@ -5,7 +5,7 @@ module SSDP
   class Consumer
     def initialize(options={})
       @options = SSDP::DEFAULTS.merge options
-      @search_socket = SSDP.create_broadcaster
+      @search_socket = SSDP.create_broadcaster options
```

Comment:
Can this be changed to `SSDP.create_broadcaster @options`, to pull in the default-merged options, in case a bind address wasn't specified when creating the consumer?

Also, can the `@options` argument be added to producer.rb, where it calls `create_broadcaster`, [here](https://github.com/daumiller/ssdp/blob/master/lib/ssdp/producer.rb#L117)?